### PR TITLE
Fix lwrp add unit tests

### PR DESCRIPTION
**Pull request by mburns**

### CODE REVIEW

File: recipes/repo.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@
   recursive true
 end
 
-apt_repo 'aptly' do
+aptly_repo 'aptly' do
```
Comment (mburns) [inline recipes/repo.rb]: [woop](https://github.com/hw-cookbooks/apt#examples)

Comment (jguice) [inline recipes/repo.rb]: is `apt_repo` a synonym for `apt_repository` ?

**Comment by jguice** [comment]
:+1:

File: recipes/repo.rb (hunk 1 above)
Comment (mburns) [inline recipes/repo.rb]: `apt_repository ` is provided by the [apt](https://github.com/hw-cookbooks/apt) cookbook.

`aptly_repo` is provided by the [aptly](https://github.com/hw-cookbooks/aptly#examples) cookbook.

`apt_repo` isn't a thing, afaik.

Comment (jguice) [inline recipes/repo.rb]: ah, shootin' straight down the middle ;)

**Pull request merged**